portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -112,15 +124,72 @@ function insertSearch() {
 
   const searchInput = document.createElement('input');
   searchInput.className = 'form-control form-inline';
+  searchInput.id = 'userSearch';
```

Comment:
I would call it "searchQuery"

Reply:
I transferred this comment to relevant repository.